### ISSUE TITLE
UniMRCP client omission of RTCP goodbye packet required by Nuance Media server for ASR  

### ISSUE THREAD

**Issue by achaloyan** [unispeech/unimrcp#50]
Originally reported on Google Code with ID 50

```
Running the unimrcpclient, connecting to Nuance Recognizer 9.0.3 with
Nuance Media Server 5.1.0 as the MRCP server causes recognition to fail
with "015 no-match-maxtime".

According to Issue 32, the "solution" is to simply modify the
incompletimeout variable, allowing Nuance to look for shorter periods of
silence to assume "end-of-speech". Indeed setting the incompletetimeout to
e.g. 900 allows recognition to work. This is clearly not an elegant solution.

According to Nuance, the correct solution is that uniMRCP should send a
"RTCP Goodbye packet" to indicate end-of-speech, otherwise their MRCP
server won't process the data.

I'm using unimrcp 0.8.0, APR 1.3.8, APR-util 1.3.9, and Sofia-SIP 1.2.10,
basically the unimrcp-deps-0.8.0.tar.gz and unimrcp-0.8.0.tar.gz files from
the unimrcp site. I'm running on CentOS 5.3 and there are no firewalls in
place between the speech server (running Nuance Recognizer, and Media
Server) and the unimrcpclient. We're using MRCP v. 2.

Any help would be appreciated.

```

Reported by `thirionjwf` on 2009-10-20 08:42:09
